packages/dds/merge-tree/src/mergeTree.ts

```diff
@@ -422,6 +422,14 @@ export interface ClientSeq {
 
 export interface IMergeTreeOptions {
 	catchUpBlobName?: string;
+	/**
+	 * Whether or not reference positions can slide to special endpoint segments
+	 * denoting the positions immediately before the start and immediately after
+	 * the end of the string.
+	 *
+	 * This is primarily useful in the case of interval stickiness.
```

Comment:
this leaves open an oddity where interval start can slide past interval end, but it's actually pretty reasonable to work around that case in an application/tell applications this can happen and to interpret it as an empty interval.

Reply:
Also linking https://dev.azure.com/fluidframework/internal/_workitems/edit/5205 here as I believe this ticket is tracking the work mentioned by this comment.